src/converter/commands/ConvertAccountsCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class ConvertAccountsCommand extends BaseCommand
+{
+    public function handle(): void
+    {
+        $servers = $this->fileService->load(FileService::SERVERS);
+        $importedServers = $this->fileService->load(FileService::IMPORTED_SERVERS);
+        $v1ProductsMap = $this->fileService->load(FileService::PACKAGES);
+
+        $v1Module = $this->db->getModule(DatabaseService::SolusVMv1);
+        $v1Products = $this->db->getProducts($v1Module);
+
+        try {
+            $initialAccounts = $this->fileService->load(FileService::HOSTBILL_ACCOUNTS);
+        } catch (\Exception $e) {
```

Comment:
Can we log this exception?

Reply:
We get exception here only if there is no proper file exist
It can be only for the first running of this command and after running this command this file appears
So it's not a problem here that this file is missing